client.go

```diff
@@ -167,12 +167,15 @@ func httpClient(ctx context.Context, addr string, namespace string, outs []inter
 		defer httpResp.Body.Close()
 
 		var resp clientResponse
-
 		if err := json.NewDecoder(httpResp.Body).Decode(&resp); err != nil {
 			return clientResponse{}, xerrors.Errorf("http status %s unmarshaling response: %w", httpResp.Status, err)
 		}
 
-		if resp.ID != *cr.req.ID {
+		if resp.ID, err = translateID(resp.ID); err != nil {
+			return clientResponse{}, xerrors.Errorf("failed to response ID: %w", err)
+		}
+
+		if resp.ID != cr.req.ID {
```

Comment:
I'm not 100% sure that comparing `interface{}` things this way is actually safe

Reply:
Should be ok. From the Go spec:

> Interface values are comparable. Two interface values are equal if they have identical dynamic types and equal dynamic values or if both have value nil.